### PR TITLE
Merge envs -> env changes to master

### PR DESCRIPTION
**Pull request by aaclan-ebi**
[ingest-central#514](https://github.com/HumanCellAtlas/ingest-central/issues/514)

### CODE REVIEW

**Comment by aaclan-ebi** [comment]
previous PR: https://github.com/HumanCellAtlas/ingest-core/pull/83

File: src/main/java/org/humancellatlas/ingest/bundle/BundleManifestRepositoryImpl.java [hunk 1]
```diff
@@ -24,7 +23,9 @@ public BundleManifestRepositoryImpl(final MongoTemplate mongoTemplate){
 
     @Override
     public Page<BundleManifest> findBundleManifestsByProjectAndBundleType(Project project, BundleType bundleType, Pageable pageable) {
-        SubmissionEnvelope submissionEnvelope = new ArrayList<>(project.getSubmissionEnvelopes()).get(0);
+        // XXX: we need to be sure if we want the submissionEnvelope this project metadocument was part of, or
+        // (as it was before) the first submission in the set of the submissionEnvelopes within this project?
```
Comment (MightyAx) [inline src/main/java/org/humancellatlas/ingest/bundle/BundleManifestRepositoryImpl.java]: Was this question resolved?

Comment (MightyAx) [reply src/main/java/org/humancellatlas/ingest/bundle/BundleManifestRepositoryImpl.java]: Can you please remove the comment before merging?

**Comment by aaclan-ebi** [comment]
This requires this state tracker change https://github.com/HumanCellAtlas/ingest-state-tracking/pull/6
Both should be released at the same time.

**Comment by aaclan-ebi** [comment]
Also requires validator change https://github.com/HumanCellAtlas/ingest-validator/commit/49d863b07b350a9cf73ea4dfc6ce21beace5df9f

**Pull request merged**